integration/test-integration/src/test/resources/apiYaml/api_policies.yaml

```diff
@@ -29,15 +29,18 @@ data:
         request:
           -
             policyName: ccAddHeader
+            policyVersion: v1
```

Comment:
The policy version, how is it applied in the YAML? For instance, can there be 2 policies with the same policy name but different policy versions? Don't we have to add int/unit tests for such scenarios, and they all are "v1" in all test policies.

Reply:
Added test for custom policies and versioned policies with https://github.com/wso2/product-microgateway/pull/2754/commits/b7f6b18870e58303c07c2b9ff89afef8a3735571
Thanks